lib/main.dart

```diff
@@ -74,6 +83,13 @@ class TMailApp extends StatelessWidget {
       unknownRoute: AppPages.unknownRoutePage,
       defaultTransition: Transition.noTransition,
       initialRoute: AppRoutes.home,
-      getPages: AppPages.pages);
+      getPages: AppPages.pages,
+    );
+  }
+
+  @override
+  void dispose() {
+    _twakeAppManager.dispose();
+    super.dispose();
```

Comment:
`TMailApp` is equal to the whole app. If this method is called, doesn't it mean the app is terminated? Why does we need to dispose this specific dependency manually?

Reply:
It is called before the application terminates. We must remove it to avoid unnecessary memory leaks when using the variable.